src/main/java/org/dice/factcheckbenchmark/benchmark/rdf/RDFResource.java

```diff
@@ -0,0 +1,85 @@
+package org.dice.factcheckbenchmark.benchmark.rdf;
+
+import org.apache.jena.rdf.model.*;
+
+import java.util.*;
+
+/**
+ * @author DANISH AHMED on 3/22/2018
+ */
+public class RDFResource {
+
+    protected Resource resource;
+    protected Model model;
+    protected String label;
+    public String uri;
+
+    private Property owlSameAsProperty = ResourceFactory.createProperty(Constants.OWL_NAMESPACE + "sameAs");
+
+    public List<Resource> owlSameAsList = new ArrayList<Resource>();     // only considering english for now
```

Comment:
it was set to List to avoid crashing, as DefactoResource.java in FactCheck has type list. I have updated the code in both factcheck-core and here to HashSet. Changes will be visible in my next push.